wiki/People/Community_Contributors/de.md

```diff
@@ -222,3 +221,17 @@ Nicht zu verwechseln mit [osu! Alumni](/wiki/People/osu!_Alumni), die frühere M
 | ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Herausragende Beiträge zur osu!-Turnierszene und den World Cups |
 | ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Herausragende Beiträge zur osu!-Turnierszene und den World Cups |
 | ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815) | Herausragende Beiträge zu osu! IRL-Events und zur Turnierszene |
+
+## Geschichte
+
+Im Februar 2018 wurde das alte Contributor-Abzeichen von ::{ flag=US }:: [RBRat3](https://osu.ppy.sh/users/307202) von ::{ flag=JP }:: [flyte](https://osu.ppy.sh/users/3103765) neu gestaltet, da es nicht zum Farbschema der neuen Webseite passte.[^redesign-reasons]
```

Comment:
For a more direct translation of `lack soul`, one could say "Fehlen einer Seele/Mangel an Seele" as well. Though I prefer what I have above.